### PR TITLE
Development

### PR DESCRIPTION
**Pull request by aledj2**
This change is [Reviewable](https://reviewable.io/reviews/moka-guys/dnanexus_tso500/1)

### CODE REVIEW

File: src/code.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		zip -r /home/dnanexus/results_temp/$sample.zip $sample
+	fi
+done
+cd /home/dnanexus/results_temp/
+# create a zip folder containing zipped folders for each sample
+zip -r /home/dnanexus/out/results_zip/Results.zip .
+cd ~
+
+# move the fastq output so they appear as seperate outputs from app for downstream tools, but go to the same place in the project
+mv /home/dnanexus/out/analysis_folder/analysis_folder/Logs_Intermediates/CollapsedReads /home/dnanexus/out/fastqs/analysis_folder/Logs_Intermediates/
```
Comment (rebeccahaines1) [inline src/code.sh:46]: shouldn't the mv new location end in CollapsedReads?

File: src/code.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# mv the bams (for coverage) so they appear as seperate outputs from app for downstream tools, but go to the same place in the project
+mv /home/dnanexus/out/analysis_folder/analysis_folder/Logs_Intermediates/StitchedRealigned /home/dnanexus/out/bams_for_coverage/analysis_folder/Logs_Intermediates/
```
Comment (rebeccahaines1) [inline src/code.sh:48]: as for line 46, should this end StitchedRealigned ?

**Review by rebeccahaines1 — CHANGES_REQUESTED** [review]
see a few comments on individual lines

**Review by aledj2 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/moka-guys/dnanexus_tso500/1)* status: 0 of 5 files reviewed, 3 unresolved discussions (waiting on @rebeccahaines1)

___
*[dxapp.json, line 56 at r1](https://reviewable.io/reviews/moka-guys/dnanexus_tso500/1#-Mp6uJj4P75xDwLrWKBn-r1-56:-Mp71gdI2gcBkS_63WX7:b-kz8mxi) ([raw file](https://github.com/moka-guys/dnanexus_tso500/blob/12bd282e80fd2e6a9a9a2f75361415f6d3d7a5e0/dxapp.json#L56)):*
<details><summary><i>Previously, rebeccahaines1 wrote…</i></summary><blockquote>

why are these two outputs indented? (lines 56-67)
</blockquote></details>

looks like a mix of tabs and spaces
___
*[src/code.sh, line 46 at r1](https://reviewable.io/reviews/moka-guys/dnanexus_tso500/1#-Mp6uJj4P75xDwLrWKBq-r1-46:-Mp71qtp4LW42tOj5QAs:b8cly3b) ([raw file](https://github.com/moka-guys/dnanexus_tso500/blob/12bd282e80fd2e6a9a9a2f75361415f6d3d7a5e0/src/code.sh#L46)):*
<details><summary><i>Previously, rebeccahaines1 wrote…</i></summary><blockquote>

shouldn't the mv new location end in CollapsedReads?
</blockquote></details>

not in this case - this command copies the CollapsedReads folder, rather than the contents. I left the CollapsedReads on the second argument the file path would be /home/dnanexus/out/fastqs/analysis_folder/Logs_Intermediates/CollapsedReads/CollapsedReads
___
*[src/code.sh, line 48 at r1](https://reviewable.io/reviews/moka-guys/dnanexus_tso500/1#-Mp6uJj4P75xDwLrWKBq-r1-48:-Mp72ddc3J-6dyX0P54X:b-5msh3o) ([raw file](https://github.com/moka-guys/dnanexus_tso500/blob/12bd282e80fd2e6a9a9a2f75361415f6d3d7a5e0/src/code.sh#L48)):*
<details><summary><i>Previously, rebeccahaines1 wrote…</i></summary><blockquote>

as for line 46, should this end StitchedRealigned ?
</blockquote></details>

as above

**Review by rebeccahaines1 — APPROVED** [review]
> Reviewed 4 of 5 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/moka-guys/dnanexus_tso500/1)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @aledj2)

**Pull request merged**